create_package.py

```diff
@@ -407,27 +453,32 @@ def main(
 
     has_client_code = bool(ADDON_CLIENT_DIR)
     if has_client_code:
-        client_dir: str = os.path.join(CLIENT_ROOT, ADDON_CLIENT_DIR)
+        if ADDON_CLIENT_DIR:
+            client_dir: str = os.path.join(CLIENT_ROOT, ADDON_CLIENT_DIR)
+        else:
+            client_dir = CLIENT_ROOT
         if not os.path.exists(client_dir):
-            raise RuntimeError(
+            msg = (
                 f"Client directory was not found '{client_dir}'."
                 " Please check 'client_dir' in 'package.py'."
             )
+            raise RuntimeError(msg)
         update_client_version(log)
 
     if only_client:
         if not has_client_code:
-            raise RuntimeError("Client code is not available. Skipping")
+            msg = "Client code is not available. Skipping"
```

Comment:
Why?

Reply:
https://docs.astral.sh/ruff/rules/raw-string-in-exception/